jetson/percep/artag_detector.cpp

```diff
@@ -127,4 +129,28 @@ cv::aruco::drawDetectedMarkers(rgb, corners, ids);
 
 double TagDetector::getAngle(float xPixel, float wPixel){
     return atan((xPixel - wPixel/2)/(wPixel/2)* tan(fieldofView/2))* 180.0 /PI;
+}
+
+void TagDetector::tagfound(rover_msgs::Target* arTags, pair<Tag, Tag> tagPair, int tag, int &buffer, Mat &depth_img, Mat &src){
```

Comment:
It's faster to copy primitives than to pass them by reference. Please change this for the bugger parameter. Can you change the tagPair to also be pass by reference since it is not a primitive?

Reply:
*buffer not bugger